src/Libraries/DSIronPython/IronPythonEvaluator.cs

```diff
@@ -104,7 +104,6 @@ private static string PythonStandardLibPath()
                 ScriptEngine PythonEngine = Python.CreateEngine();
                 if (!string.IsNullOrEmpty(stdLib))
                 {
-                    code = "import sys" + System.Environment.NewLine + code;
```

Comment:
why was this not required?

Reply:
Looking at the code, it did not make sense to me. I removed it and made sure the std lib is still loaded in the test.

Reply:
This was being imported by default into all the Python scripts before. But If we remove it completely, I think this would break in the case where the users do not explicitly have this "import sys" in their code but try to access the "sys" data. They could just use sys.path without actually having to import.